src/networking.c

```diff
@@ -1441,6 +1441,10 @@ void unlinkClient(client *c) {
                 }
             }
         }
+        /* Only use shutdown when the fork is active and we are the parent. */
+        if (server.child_type && !server.in_fork_child) {
```

Comment:
server.child_type is only set in the parent.
there's no need for `in_fork_child` check in the condition.

Reply:
yes, before i just wanted to make the code explicit, i will remove the `in_fork_child` check